tests/test_e2e_10_mef_eline.py

```diff
@@ -489,23 +506,89 @@ def test_045_create_circuit_reusing_same_vlanid_from_previous_evc(self):
+                    "value": 101
+                }
+            },
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:2"}},
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:02:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:2"}}
+            ],
+            "backup_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:4"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:3"}}
+            ]
+        }
 
-    def test_055_patch_evc_new_unis(self):
-        # TODO
-        assert True
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        assert response.status_code == 201
 
-    def test_060_disable_evc(self):
-        # TODO
-        assert True
+        time.sleep(10)
 
-    def test_065_on_primary_path_fail_should_migrate_to_backup(self):
-        # TODO
-        assert True
+        # Check on the virtual switches directly for flows
+        s1, s2, s3 = self.net.net.get('s1', 's2', 's3')
+        flows_s1 = s1.dpctl('dump-flows')
+        flows_s2 = s2.dpctl('dump-flows')
+        flows_s3 = s3.dpctl('dump-flows')
 
-    def test_070_delete_evc_after_restart_kytos_and_no_switch_reconnected(self):
+        assert len(flows_s1.split('\r\n ')) == 3
+        assert len(flows_s2.split('\r\n ')) == 3
+        assert len(flows_s3.split('\r\n ')) == 3
+
+        # Command to up/down links to test if back-up path is taken
+        self.net.net.configLinkStatus('s1', 's2', 'down')
+
+        # Wait just a few seconds to give time to the controller receive and process the linkDown event
+        time.sleep(10)
+
+        # Nodes should be able to ping each other
+        h11, h3 = self.net.net.get('h11', 'h3')
+        h11.cmd('ip link add link %s name vlan101 type vlan id 101' % (h11.intfNames()[0]))
+        h11.cmd('ip link set up vlan101')
+        h11.cmd('ip addr add 101.0.0.1/24 dev vlan101')
+        h3.cmd('ip link add link %s name vlan101 type vlan id 101' % (h3.intfNames()[0]))
+        h3.cmd('ip link set up vlan101')
+        h3.cmd('ip addr add 101.0.0.3/24 dev vlan101')
+        result = h11.cmd('ping -c1 101.0.0.3')
+        assert ', 0% packet loss,' in result
+
+        # Clean up
+        h11.cmd('ip link del vlan101')
+        h3.cmd('ip link del vlan101')
+
+        # # Check on the virtual switches directly for flows
+        flows_s1 = s1.dpctl('dump-flows')
+        flows_s2 = s2.dpctl('dump-flows')
+        flows_s3 = s3.dpctl('dump-flows')
+
+        assert len(flows_s1.split('\r\n ')) == 3
+        assert len(flows_s2.split('\r\n ')) == 1
+        assert len(flows_s3.split('\r\n ')) == 3
+
+        # Command to up/down links to test if back-up path is taken
+        self.net.net.configLinkStatus('s1', 's2', 'up')
```

Comment:
Please move this a little above, right before the first assertion (line 570). Otherwise, in case of the assertion fail it can impact other tests.

Reply:
This is resolved with commit [e8f260c](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/e8f260c10d8b078f233524c19e76606ce3ec0d71)